src/services.js

```diff
@@ -47,8 +47,11 @@ export function isOutsideOfBusinessHours(currentDatetime, holidays) {
  *  Transfer the incoming call to the overflow number
  */
 export async function transferCall(callObject) {
-  // Do nothing if the Webhook event is not associated to the main number:
-  if (callObject.number.id !== PHONE_NUMBER_AIRCALL_IDS.MAIN) {
+  // Do nothing if the Webhook event is not associated to the main number or
+  // the Toronto brokerage number:
```

Comment:
The comment should probably be made more generic to match the code, e.g.
```suggestion
  // Do nothing if the Webhook event is not associated with a number we are
  // configured to forward:
```